README.md

```diff
@@ -128,9 +132,13 @@ Do a `hab svc status` to check the status of all the services. They may take a f
 
 If things don't work as expected (eg, if all the services are not in the `up` state), please see the Troubleshooting section below.
 
-## Web UI
+## Minio Web UI
+
+The On-Premise Depot now stores package artifacts in Minio (https://github.com/minio/minio). By default, the Minio instance will be available on port 9000 (or whatever port you specified in your `bldr.env`). Visit http://{YOUR_MINIO_ENDPOINT} to confirm that the Minio UI is available, and that you can log in with the credentials that were specified in your `bldr.env` file. There should already be a bucket created in which to host the artifacts.
```

Comment:
I'd remove the first instance of "now" from this sentence, and just say the depot stores package artifacts in Minio.

Reply:
It's also not obvious from these instructions what `YOUR_MINIO_ENDPOINT` is, even though we know it's just the IP/hostname of whatever machine this is being installed to.  It might be worth mentioning that.

Reply:
Cool, will fix